### PR TITLE
{bio}[GCC/10.3.0] vt v0.57721

### PR DESCRIPTION
**Pull request by pberthier**
(created using `eb --new-pr`)

edit: requires:
* #15641
* #15642
* #15645

### CODE REVIEW

**Comment by pberthier** [comment]
Test report by @pberthier
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
softdev.binf.unibe.ch - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz, Python 3.6.8
See https://gist.github.com/ec968ccb11a6d34379bee9c922b54892 for a full test report.

File: easybuild/easyconfigs/v/vt/vt-0.57721-GCC-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+# checksums = [
+#     '8f06d464ec5458539cfa30f81a034f47fe7f801146fe8ca80c14a3816b704e17',  # 0.57721.tar.gz
+#     '5a31713ae7c5febf850eec294dbba45b6b240d832cb94bdfe9ee695fd13e45bb',  # vt-0.57721_use-external-libs.patch
+#     '663525f5180a66d8817f271a69de1884576e2414d46b7ba93029d78f71d5b14c',  # vt-0.57721_fix-Rmath-includes.patch
+#     '56d874d5c7d488761f25afb4ee3717c88dfd959ed76efb75a8531b8428d30ccd',  # vt-0.57721_avoid-implicit-conversion.patch
+#     'ba2cfc24a88a4291b2da634db49e18199c379eb95d3b3acac59556bc68c292df',  # vt-0.57721_update-faidx.patch
+#     '4146282eb6ec852485f5a16e647da9c3b210450ceb5519b9b4a98bc52124469a',  # vt-0.57721_fail-on-test-error.patch
+# ]
+
```
[the remark this answers is not in the document]
Comment (pberthier) [reply easybuild/easyconfigs/v/vt/vt-0.57721-GCC-10.3.0.eb:41]: Thanks, I'll do that

**Comment by boegel** [comment]
@pberthier All required dependencies are merged, now this needs a sync with the `develop` branch (easy to do via `eb --sync-pr-with-develop 15646`)

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 5 out of 5 (1 easyconfigs in total)
node080.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/3c4f9dcbfb8ed50506f9dd019241b460 for a full test report.

**Comment by jfgrimm** [comment]
@boegelbot please test @ generoso

**Comment by jfgrimm** [comment]
@boegelbot please test @ jsc-zen2

**Pull request merged**